crates/smdk/src/build.rs

```diff
@@ -0,0 +1,45 @@
+use anyhow::{Error, Result};
+use clap::Parser;
+use std::process::Command;
```

Comment:
use our standard import format which is: std, third party,  internal dependencies